airflow/www/static/js/utils/index.ts

```diff
@@ -25,6 +25,8 @@ import useOffsetTop from "./useOffsetTop";
 // Delay in ms for various hover actions
 const hoverDelay = 200;
 
+const logMarkers = ["::group::", "::endgroup::"];
```

Comment:
The markers are inconsistent to the origin definition in airflow/www/static/js/dag/details/taskInstance/Logs/utils.ts:83

As the original marker is a regex, I'd recommand to (1) consolidate the definition / pass the marker in the function and also use a regex and not a direct match

Reply:
I have updated `highlightByKeywords` method to get the log marker regex defined in the original place and modified the function to do regex matching.